builds.go

```diff
@@ -0,0 +1,293 @@
+package main
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+
+	"github.com/docker-library/meta-scripts/om"
+
+	c8derrdefs "github.com/containerd/containerd/errdefs"
+	"github.com/docker-library/bashbrew/registry"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/sirupsen/logrus" // this is used by containerd libraries, so we need to set the default log level for it
+)
+
+var concurrency = 50
```

Comment:
Why `var` and not `const`?

Reply:
Just because I moved it up so it'd be more obvious and didn't give it much thought -- I ultimately wanted to make it a CLI flag, but changing it doesn't really change very much (it makes the whole thing slightly faster or slower, but still generally less than a minute so still subject to / limited by the 429s which are based on requests per minute). :sweat_smile: